README.md

```diff
@@ -295,16 +398,380 @@ Here is an example:
+        \"address\": { },
+        \"name\": {
+          \"first\": \"John\",
+          \"last\": \"Johnson\"
+        }
+      },
+      \"job\": {
+        \"company\": \"Good Inc.\",
+        \"title\": \"CEO\",
+        \"location\": {
+          \"country\": \"Norway\",
+          \"city\": \"Oslo\"
+        }
+      },
+      \"skill-refs\": [
+        \"skill-1\",
+        \"skill-3\"
+      ]
+    },
+    {
+      \"personal\": {
+        \"address\": { },
+        \"name\": {
+          \"first\": \"Sammy\",
+          \"last\": \"Smith\"
+        }
+      },
+      \"job\": {
+        \"company\": \"Good Inc.\",
+        \"title\": \"VP, Marketing\",
+        \"location\": {
+          \"country\": \"USA\",
+          \"city\": \"Peoria\"
+        }
+      },
+      \"skill-refs\": [
+        \"skill-2\",
+        \"skill-3\"
+      ]
+    }
+  ],
+  \"skills\": {
+    \"skill-1\": {
+      \"name\": \"Programming\",
+      \"description\": \"Can code like Richard S or Linus T\"
+    },
+    \"skill-2\": {
+      \"name\": \"Design\",
+      \"description\": \"Can think out loud, structure code systems\"
+    },
+    \"skill-3\": {
+      \"name\": \"Communication\",
+      \"description\": \"Can explain complicated things to social scientists\"
+    }
+  }
+}")
+```
+
+The code below demonstrates how to use the `skills` hash table and
+symbols as keys:
+
+```emacs-lisp
+(let* ((people-data (jeison-read jeison-people json-people)))
+    ;; Do useful things with the data (in this case, just print):
+    (mapcar (lambda (person)
+              (princ (format "%s skills: %s\n"
+                             (oref person name)
+                             (mapconcat (lambda (the-skill-ref)
+                                          (let ((the-skill (gethash the-skill-ref
+                                                                    (oref people-data skills))))
+                                            (oref the-skill name)))
+                                        (oref person skill-refs)
+                                        ", "))))
+            (oref people-data people))
+    (pp (oref people-data skills)))
+```
+
+The output in the *ielm* buffer should be:
+
+```
+Johnson skills: Programming, Communication
+Smith skills: Design, Communication
+#s(hash-table size 65 test eq rehash-size 1.5 rehash-threshold 0.8125 data
+              (skill-1 #s(jeison-skill "Programming" "Can code like Richard S or Linus T")
+                       skill-2 #s(jeison-skill "Design" "Can think out loud, structure code systems")
+                       skill-3 #s(jeison-skill "Communication" "Can explain complicated things to social scientists")))
+```
+
+The contents of the `skills` hash table are pretty-printed to show the
+keys are symbols.
+
+`jeison-hash-table-of` also accepts `string` for the hash table's key
+type. Simply change the definitions of `jeison-people` and
+`jeison-person` to use `string` instead of `symbol`:
```

Comment:
This and the examples below don't add a lot of value, but bloat the documentation making hash tables look more complicated than they actually are. I suggest cutting all and keep only the information that we could've also used `string` type there. Maybe even rephrase it and explicitly say that it accepts ONLY strings and symbols as keys.

Reply:
I still believe that having two almost identical examples is unnecessary for understanding. It bloats the documentation and makes this feature seem more complex than it is.